docs/API_KEY.md

```diff
@@ -0,0 +1,112 @@
+# How to add new/own API key to tinode?
+
+## What is API key?
+
+Trinode encrypts data for security. For this purpose, two keys are used:
+
+* `HMAC salt` in server side. Hereafter referred to as _Server Key_.
+* `API key` in client side. Hereafter referred to as _Client Key_.
+
+## Prerequisite

```

Comment:
This section is not needed. Run `keygen --salt=auto` and it will auto-generate cryptographically-secure random salt.